allure/pytest_plugin.py

```diff
@@ -132,6 +133,14 @@ def attach(self, title, contents, attach_type):
                         type=attach_type)
         self.stack[-1].attachments.append(attach)
 
+    def dynamic_issue(self, *issues):
+        """
+        Attaches ``issues`` to the current active case
+        """
+        if self.test:
+            issues = [TestLabel(name=Label.ISSUE, value=issue) for issue in issues]
+            self.test.labels.extend(issues)
```

Comment:
To amuse my perfectionism and win us a "line saver of the month" award please move the comprehension inside of the `extend()` call so we dont waste our precious bytes on .. stuff.